docs/getting-started/index.md

```diff
@@ -14,15 +20,19 @@ If you are not sure where you should get started, check out our **Guided User St
 
 ## Products
 
-Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a high level overview of the different products which comprise these space:
+Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a quick overview of the different products which comprise these spaces:
 
-* [Substrate](https://github.com/paritytech/substrate): A next generation blockchain development framework built using Rust and Wasm.
-* [ink!](https://github.com/paritytech/ink): A Rust based eDSL for building Wasm smart contracts compatible with Substrate's Contract module.
+* [Substrate](https://github.com/paritytech/substrate): A next generation blockchain development framework built using WebAssembly, Libp2p and GRANDPA Consensus.
+* [ink!](https://github.com/paritytech/ink): A Rust based eDSL for building WebAssembly based smart contracts compatible with Substrate's Contract module.
 * [Polkadot-JS/API](https://github.com/polkadot-js/api): Promise and RxJS APIs for interacting with Substrate-based chains via RPC calls.
 * [Polkadot-JS/Apps](https://github.com/polkadot-js/apps): A basic UI for interacting with all features of a Substrate node. Powered by Polkadot-JS/API.
-* [oo7-substrate (Bonds Library)](https://github.com/paritytech/oo7/tree/master/packages/oo7-substrate): A ReactJS based API for interacting with Substrate-based chains via RPC Calls.
+* [oo7-substrate](https://github.com/paritytech/oo7/tree/master/packages/oo7-substrate): A ReactJS based API for interacting with Substrate-based chains via RPC Calls.
 * [Substrate UI](https://github.com/paritytech/substrate-ui): A basic ReactJS based UI built for quickly building new interfaces for Substrate chains. Powered by oo7-substrate.
 
+## Reference documents
```

Comment:
I think it makes sense to have this as the first line since users may be looking for this when clicking docs. They should not have to read all the other lines to get where they want to go.

Reply:
Done, move it to the first section on this page.